lighthouse-cli/ask.ts

```diff
@@ -20,7 +20,7 @@
 const readline = require('readline');
 
 function ask(question: string, options: string[]): Promise<string> {
-  return new Promise((resolve, reject) => {
+  return new Promise((resolve) => {
```

Comment:
to be consistent with the rest of the codebase, can you remove the brackets for the single param?